src/platform/packages/shared/response-ops/alerts-fields-browser/components/field_items/field_items.tsx

```diff
@@ -67,29 +66,35 @@ export const getFieldItemsData = ({
     selectedCategoryIds.length > 0 ? selectedCategoryIds : Object.keys(browserFields);
   const selectedFieldIds = new Set(columnIds);
 
-  const fieldItems = uniqBy(
-    'name',
-    categoryIds.reduce<BrowserFieldItem[]>((fieldItemsAcc, categoryId) => {
+  const getFieldItems = () => {
+    const fieldItemsAcc: BrowserFieldItem[] = [];
+    const fieldSeen: Map<string, boolean> = new Map();
```

Comment:
nit: maybe Set would be better, since only value for set for Map key would be `true` only